x-pack/plugins/ingest_manager/public/applications/ingest_manager/hooks/use_package_icon_type.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect, useMemo, useState } from 'react';
+import { ICON_TYPES } from '@elastic/eui';
+import { PackageInfo, PackageListItem } from '../../../../common/types/models';
+import { useLinks } from '../sections/epm/hooks';
+
+type Package = PackageInfo | PackageListItem;
+
+export interface UsePackageIconType {
+  packageName: Package['name'];
+  version?: Package['version'];
+  icons?: Package['icons'];
+}
+
+const CACHED_ICONS = new Map<string, string>();
+
+export const usePackageIconType = ({
+  packageName = '',
+  version = '',
+  icons = [],
+}: UsePackageIconType) => {
+  const { toImage } = useLinks();
+  const [iconType, setIconType] = useState<string>(''); // FIXME: use `empty` icon during initialization - see: https://github.com/elastic/kibana/issues/60622
```

Comment:
Hi @jfsiii I think the EUI bug fix is in Master now, so this can be fixed here by setting the initial icon type to `empty` so that there is still an icon "placeholder" shown on the UI.
Could you make that change and mark this PR as closing out #60622 ?
Change would be:
```suggestion
  const [iconType, setIconType] = useState<string>('empty');
```

Reply:
I looked into that, it's in EUI master but not yet in Kibana master https://github.com/elastic/kibana/pull/62065

Reply:
ahh. Cool. Thanks. I had seen a few attempts at getting the builds to work but then stoped getting those notifications, so I thought it was in :)